genesyscloud/organization_authentication_settings/genesyscloud_organization_authentication_settings_init_test.go

```diff
@@ -0,0 +1,47 @@
+package organization_authentication_settings
```

Comment:
@carnellj Do we need an init_test file if there are no integration tests?

Reply:
No.  We can add one at a later date if we do not need one.